[codestyle] plugins xml

brianteeman opened the pull request:
codestyle review for the xml files in the plugins folder

brianteeman committed:
[codestyle] plugins xml

Quy commented on plugins/authentication/ldap/ldap.xml:
```diff
@@ -61,30 +61,30 @@
 					label="PLG_LDAP_FIELD_REFERRALS_LABEL"
 					description="PLG_LDAP_FIELD_REFERRALS_DESC"
-				>
+					default="0"
```
Swap lines `class` before `default`.

brianteeman replied:
the codestyle guide only specifies the order of name/type/label/description

wojsmol commented:
I'll review later.
cc @andrepereiradasilva Plase review.

Quy commented on plugins/editors/tinymce/form/setoptions.xml:
```diff
@@ -1,313 +1,313 @@
+        multiple="true"
+        class="access-select"
+        label="PLG_TINY_FIELD_SETACCESS_LABEL"
+        description="PLG_TINY_FIELD_SETACCESS_DESC"
```
Move `label` and `description` below `type`.

wojsmol replied:
@Quy See https://github.com/brianteeman/joomla-cms/blob/4de9fb6226d68276c7797e8fb270e28239e73c2b/plugins/editors/tinymce/form/setoptions.xml this ok

brianteeman replied:
@wojsmol I fixed this earlier this morning

brianteeman committed:
missed one - thanks @quy

Quy commented:
Reformat https://github.com/brianteeman/joomla-cms/blob/fd13b736c1d4e1794dc6b152b319c4eb0c4e28e1/plugins/editors/tinymce/tinymce.xml#L28

brianteeman committed:
Update tinymce.xml

brianteeman commented:
@quy did that fix

brianteeman committed:
Merge branch 'staging' into xml-plugins

brianteeman commented:
conflicts resolved

zero-24 commented on plugins/fields/sql/sql.xml:
```diff
@@ -28,6 +28,7 @@
 					description="PLG_FIELDS_SQL_PARAMS_QUERY_DESC"
 					filter="raw"
 					rows="10"
+					filter="raw"
```
Filter is dublicated here

Commit:
Update article.xml (#16256)

* Update article.xml

* addtional XML CS fixes

* part 2

brianteeman committed:
grrg

brianteeman commented:
@zero-24 fixed that

Quy commented:
In profile.xml, add a line before lines 274 and 295.

brianteeman committed:
new lines

brianteeman commented:
@quy done

Quy commented:
I have tested this item :white_check_mark: successfully on f07e24035a599dfd1ce72cc4bd60c6325b536d5c<br /><br />Code review.<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/15932">issues.joomla.org/tracker/joomla-cms/15932</a>.</sub>

brianteeman committed:
[a11y] misc admin modules (#15068)

* logout button on mod_logged

* multilingual status icon

* admin stats modules line icons

* admin status module

* home icon on default menu

* update

I went for the `<span class="element-invisible">` instead of the aria-title so it is more consistent with other uses elsewhere

brianteeman committed:
Merge branch 'staging' into xml-plugins

brianteeman commented:
Updated branch to restart travis - no need for new tests

rdeutz committed:
Merge branch 'staging' into xml-plugins

andrepereiradasilva commented:
I have tested this item :white_check_mark: successfully on ddcaf07b54bdd49d8a1e1a31b2f79debc8923a9d<br /><br />ok on code review.<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/15932">issues.joomla.org/tracker/joomla-cms/15932</a>.</sub>

brianteeman commented:
RTC please?

franz-wohlkoenig commented:
RTC

brianteeman commented:
@wilsonge you will probably want to merge this and #15922 before you do your 3.8->4 merge

Quy commented:
@brianteeman The following have blank lines to delete. I attempted to submit patches but they contain previous commits. Can you remove them or tell me how to do via Github interface?  Thanks.

plugins/authentication/cookie/cookie.xml
plugins/authentication/ldap/ldap.xml
plugins/content/emailcloak/emailcloak.xml
plugins/editors/tinymce/tinymce.xml
plugins/fields/textarea/params/textarea.xml
plugins/fields/textarea/textarea.xml

zero-24 commented:
@quy do you have Glip? If yes please contace me and i can help you with github. If you do not have a account please send me a mail to tobias.zulauf[at]community.joomla.org

Commit:
Update textarea.xml (#62)

Commit:
Update textarea.xml (#61)

Commit:
Update tinymce.xml (#60)

Commit:
Update emailcloak.xml (#59)

Commit:
Update ldap.xml (#58)

brianteeman commented:
Thanks
